### PR TITLE
chore: wrap config response

### PR DESCRIPTION
**Pull request by hoangharry**
**What does this PR do?**

-   [x] Wrap config response

**Feature affected:**
- default role
- nft role
- log
- welcome
- starboard

### CODE REVIEW

File: pkg/handler/configs.go [hunk 1]
```diff
@@ -90,18 +90,18 @@ func (h *Handler) GetWelcomeChannelConfig(c *gin.Context) {
 	guildID := c.Query("guild_id")
 	if guildID == "" {
 		h.log.Info("[handler.GetWelcomeChannelConfig] - guild id empty")
-		c.JSON(http.StatusBadRequest, gin.H{"error": "guild_id is required"})
+		c.JSON(http.StatusBadRequest, response.CreateResponse[any](nil, nil, errors.New("guild_id is required"), nil))
 	}
```
Comment (nguyennh4522) [inline pkg/handler/configs.go:94]: did you miss return?

File: pkg/handler/configs.go [hunk 2]
```diff
@@ -123,21 +123,21 @@ func (h *Handler) UpsertWelcomeChannelConfig(c *gin.Context) {
...
 	if req.GuildID == "" {
 		h.log.Info("[handler.GetWelcomeChannelConfig] - guild id empty")
-		c.JSON(http.StatusBadRequest, gin.H{"error": "guild_id is required"})
+		c.JSON(http.StatusBadRequest, response.CreateResponse[any](nil, nil, errors.New("guild_id is required"), nil))
 	}
```
Comment (nguyennh4522) [inline pkg/handler/configs.go:127]: did you miss return?

File: pkg/handler/configs.go [hunk 3]
```diff
@@ -123,21 +123,21 @@ func (h *Handler) UpsertWelcomeChannelConfig(c *gin.Context) {
...
 	if req.ChannelID == "" {
 		h.log.Info("[handler.GetWelcomeChannelConfig] - channel id empty")
-		c.JSON(http.StatusBadRequest, gin.H{"error": "channel_id is required"})
+		c.JSON(http.StatusBadRequest, response.CreateResponse[any](nil, nil, errors.New("channel_id is required"), nil))
 	}
```
Comment (nguyennh4522) [inline pkg/handler/configs.go:131]: same as above

File: pkg/handler/configs.go [hunk 4]
```diff
@@ -159,16 +159,16 @@ func (h *Handler) DeleteWelcomeChannelConfig(c *gin.Context) {
...
 	if req.GuildID == "" {
 		h.log.Info("[handler.GetWelcomeChannelConfig] - guild id empty")
-		c.JSON(http.StatusBadRequest, gin.H{"error": "guild_id is required"})
+		c.JSON(http.StatusBadRequest, response.CreateResponse[any](nil, nil, errors.New("guild_id is required"), nil))
 	}
```
Comment (nguyennh4522) [inline pkg/handler/configs.go:163]: same as above

**Review by trkhoi — APPROVED** [review]

**Review by nguyennh4522 — APPROVED** [review]

**Pull request merged**